### ISSUE TITLE
SDL2.dll is missing

### ISSUE THREAD

**Issue by MartenBE** [mupen64plus/mupen64plus-ui-python#93]
I've installed m64py on windows 10 by using the installer from sourceforge (version 0.2.4). The program starts up, but crashes as soon as I try to load a ROM. I get a black window and before the N64 logo comes up, the application quits without notification.

When I start mupen64plus.exe on the commandline, a notification pops up saying:

> The program cam't start because SDL2.dll is missing from your computer. Try reinstalling the program to fix this problem.

I did reinstall, but the error persists. Also, I can see there is an SDL2.dll in the lib folder.

**Comment by gen2brain** [comment]
You tried only one game with one video plugin or it crashes every time? 

You cannot start mupen64plus.exe unless you copy SDL2.dll to same dir, only python app knows about libraries in lib directory.

**Comment by MartenBE** [comment]
I've only tried one game with one plugin. I will try copying the dll files into the main folder.

**Comment by MartenBE** [comment]
Copying the dll into the main folder from the <main>/lib folder does not help. I've tried it with a fresh install and it crashes on both Ocarina of Time and Majora's Mask... I tried to read the log, but the logviewer closes with the application. Is there anyway to read the log file outside m64py?

**Comment by gen2brain** [comment]
v0.2.5 is released, try with the latest version, I will close this one. Feel free to open a new issue if it is not resolved. Btw. you can now use setup-console.exe, so you can see the logs from cmd.
